plugins/Number.py

```diff
@@ -59,19 +75,28 @@ def node(self, data, tags):
                 m = self.Number.match(tags[i])
                 if (not m and
                     not (i == "width" and tags[i] == "narrow") and
-                    not (i == "maxspeed" and (
+                    not (("maxspeed" in i or "minspeed" in i) and (
                         tags[i] in self.MaxspeedExtraValue or
                         self.MaxspeedClassValue.match(tags[i]) or
                         (tags[i] == "implicit" and ("traffic_sign" in tags) and "maxspeed" in tags["traffic_sign"].split(";"))
                     )) and
                     not (i == "maxheight" and tags[i] in self.MaxheightExtraValue)
                 ):
                     return {"class": 3091, "subclass": 1, "text": T_("Concerns tag: `{0}`", '='.join([i, tags[i]])) }
-                elif m and i == "height" and float(m.group(1)) > 500:
+                if not m:
+                    continue
+
+                # Below here only tags containing numbers with/without unit remain
+                if i in self.tag_number_integer and str(int(abs(float(m.group(1))))) != tags[i]:
```

Comment:
`float([ramdom])` raise an exception. You cannot use it like this to check for interger. You have to catch ValueError.

Reply:
"match group 1" of the regex is: `((?:-?[0-9]+(?:[.][0-9]+)?)|(?:[.][0-9]+))`
If I'm not mistaken, that should always end up with either:
- `[number(s)]` (e.g. `5`)
- `[number(s)][dot][number(s)]` (e.g. `22.3`)
- `[dot][number(s)]` (e.g. `.93`)

Which are all float-parsable. I don't see any case that could throw a ValueError?
(It's different for `int()`, which will throw in case I give a float string as input, i.e. `int("3.0")` throws a `ValueError`. Therefore I first use `float()` inside `int()`)

Would you still prefer that I make it 1) a separate function and 2) wrap it in try/catch? No problem of course :)

Reply:
You are right. It is ok.

Reply:
Thank you for double-checking though :) !